docs/provision.md

```diff
@@ -101,6 +101,22 @@ You can set the number of nodes and virtual machine type for each component
 of the architecture. Take the next example as reference for a large project
 but adapt them to your necessities.
 
+Special attention to `opensearch_dashboards_node_count` and `opensearch_dashboards_anonymous_node_count`.
```

Comment:
This section needs to be reworded. It's not clear what the purpose of it is. It needs a better introduction. I can work on something.